python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/router_isis.schema.yml

```diff
@@ -393,3 +368,30 @@ keys:
       eos_cli:
         type: str
         description: Multiline EOS CLI rendered directly on the router isis in the final EOS configuration.
+      address_family:
+        type: list
+        items:
+          type: str
+          valid_values:
+            - "ipv4"
+            - "ipv6"
+            - "ipv4 unicast"
+            - "ipv6 unicast"
+          description: Address Family.
+        deprecation:
+          removed: true
+          warning: true
+          new_key: address_family_ipv4.enabled or address_family_ipv6.enabled
+          remove_in_version: "5.0.0"
+      isis_af_defaults:
+        type: list
+        items:
```

Comment:
Please remove the items key and everything below, since it will only take up space in the doc table.

Reply:
Fixed!